docs/lib/datastore/fragments/ios/data-access/query-predicate-multiple-snippet.md

```diff
@@ -1,6 +1,7 @@
 ```swift
 let p = Post.keys
-Amplify.DataStore.query(Post.self, where: { p.rating > 4 && p.status == .active }) {
+let predicate = p.rating > 4 && p.status == .active
+Amplify.DataStore.query(Post.self, where: predicate) {
```

Comment:
can you keep the condition inline?

Reply:
woops, some of the errors got me mixed up what was valid syntax. added back inline